util/SqliteMigrations/HelperScripts/2023-12-04_00_Up_GrantIndexes.sql

```diff
@@ -0,0 +1,46 @@
+﻿ALTER TABLE
+"Grant" RENAME TO "Old_Grant";
+
+CREATE TABLE "Grant"
+(
+  "Id" INTEGER PRIMARY KEY AUTOINCREMENT,
+  "Key" TEXT NOT NULL,
+  "Type" TEXT NOT NULL,
+  "SubjectId" TEXT NULL,
+  "SessionId" TEXT NULL,
+  "ClientId" TEXT NOT NULL,
```

Comment:
I don't believe this is a thing in SQLite.

Reply:
I believe you can declare length but it will not enforce it, sqllite will allow you to store any datatype in any column regardless of the declared data type with the exception of integer primary key